Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Last-Update: 2012-03-06
Bug-Debian: http://bugs.debian.org/443774
Forwarded: https://bugzilla.gnome.org/show_bug.cgi?id=671444
Applied-Upstream: r22100, 2.5.0
Description: Suppress worthless console message.
 Gnucash print worthless and confusing message to console on start.
 This is not intended to be used by users and it unnecessarily
 worries people who are likely to take it as warning rather than
 normal informational message.
 It has been reported as especially annoying if gnucash is running
 as cron job because it requires special workaround against sending
 this message over email on each invocation.
 .
 This patch suppresses printing the message.


--- a/src/bin/gnucash-bin.c
+++ b/src/bin/gnucash-bin.c
@@ -816,8 +816,6 @@
             g_error_free(binreloc_error);
         }
     }
-#else
-    g_message("main: binreloc relocation support was disabled at configure time.\n");
 #endif
 
     /* This should be called before gettext is initialized
